pages/api/subscribe.ts

```diff
@@ -0,0 +1,127 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import md5 from "md5";
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+export default async function handler(req: NextApiRequest, res: NextApiResponse): Promise<any> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I really didn't feel typing the whole return data here haha. I'll cleanup that in the next PR.

Reply:
No need - we don't have to be too rigid on the TS stuff for the purposes of this site haha